6-anthos-install/db-scripts/populate-accounts-db.yaml

```diff
@@ -0,0 +1,342 @@
+# Copyright 2021 Google LLC
```

Comment:
Could we just point to the upstream repo or did we make any modifications to this?

Reply:
@bharathkkb 
we changed the images: 

- image: gcr.io/google-containers/alpine-with-bash:1.0
- image: gcr.io/cloud-marketplace/google/postgresql13

so that they are fetch from  `gcr.io`

and the postgresql13 part was missing:

```
resources:
  limits:
    cpu: "200m"
    memory: "100Mi"
```